runtime/src/debug/cpp/SourceInfo.cpp

```diff
@@ -156,6 +156,7 @@ extern "C" struct SourceInfo Kotlin_getSourceInfo(void* addr) {
         uint32_t lineNumber = CSSourceInfoGetLineNumber(ref);
         if (lineNumber == 0)
           return 0;
+        RuntimeAssert(lineNumber <= static_cast<uint32_t>(std::numeric_limits<int32_t>::max()), "Unexpectedly huge line number");
```

Comment:
I'm not sure we should have such assertions in this code.
Motivation: on the one hand, the entire approach used in this code is fragile, so we actually can get somewhat incorrect data. On the other hand, this code is used in error handling. If we allow incorrect data, it will be displayed in stack traces, giving us more information than if we just crash here.

Reply:
Makes sense, removed the asserts.